app/src/main/java/com/example/composetemplate/data/models/local_models/User.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.composetemplate.data.models.local_models
+
+import com.example.composetemplate.utils.DateUtil
+import kotlinx.serialization.Serializable
+
+// here we don't save password security wise
+@Serializable
+data class User(
+    var permission_type: String,
+    var full_name: String,
```

Comment:
There is special reason to use snake case instead of camel case?

Reply:
Note to my self: check if @Serialize... is good to push as body